### DISCUSSION TITLE
Is it possible to monitor game servers via IP:port?

### DISCUSSION THREAD

**Discussion by SebaYB** [upptime/upptime#724] [Q&A] [answered]
What the title says, if possible I would like to know how to implement it.

**Comment by ErdbeerbaerLP** [comment, marked as answer]
At least minecraft servers do work, unsure about others. Feel free to copy from [my config](https://github.com/ErdbeerbaerLP/status-page/blob/74600b19c2ceecc08ba45b29769f52427a664337/.upptimerc.yml#L19-L23) and see how I did it.
